gridfinity-rebuilt-utility.scad

```diff
@@ -268,7 +271,7 @@ module gridfinityBase(gx, gy, length, dx, dy, hole_options=bundle_hole_options()
  * @param off
  * @param size [x, y] size of a single base.  Only set if deviating from the standard!
  */
-module block_base(hole_options, off=0, size=[BASE_SIZE, BASE_SIZE]) {
+module block_base(hole_options, off=0, size=[BASE_SIZE, BASE_SIZE], thumbscrew=false) {
```

Comment:
Don't forget to add the additional parameter as a comment, and an assertion to ensure the value being passed in is a bool.